content/tools/cli/Cli-tool.md

```diff
@@ -0,0 +1,107 @@
+<!--
+title: "CLI Tool"
+description: "Instructions for installing the CLI tool for library analysis"
+tags: "tools cli library install"
+-->
+  
+Use Contrast's command line interface (CLI) tool for software composition analysis (SCA) to gather and send library dependency information for your applications to the Contrast UI. 
+
+## How It Works
+
+The Contrast SCA CLI tool works in tandem with the Contrast UI to help you and your organization secure your application. For the Contrast SCA CLI tool to work with the UI, it must identify itself with a set of credentials, and also identify the application of which it's providing analysis.  
+
+The Contrast SCA is invoked by a command line interface. 
+
```

Comment:
Can we add a couple more sentences here about how the tool works?

Reply:
How about adding
"The Contrast SCA CLI tool works in tandem with the Contrast UI to help you and your organization secure your application by giving you the visibility to understand how vulnerable libraries got introduced. When an open source library is added to an application, all of it’s dependencies are also inherited. Some of those indirect dependencies could be introducing vulnerabilities into your applications.  The Contrast SCA tool will identify all open source libraries and visualize these libraries as a hierarchical dependency tree.

For the Contrast SCA CLI tool to work with the UI, it must identify itself with a set of credentials, and also identify the application of which it's providing analysis.  "

Reply:
Incorporated some of this into the revised section here - some of it I saved for the library hierarchy article.